README.md

```diff
@@ -79,10 +79,42 @@ npx yarn <<<my commands here>>>
 git clone git@github.com:dso-toolkit/dso-toolkit.git
 cd dso-toolkit
 yarn install
-yarn start
 ```
 
-A local webserver is started at http://localhost:3000/. See the [Fractal guide](https://fractal.build/) on how to develop components. See [CONTRIBUTING.md](CONTRIBUTING.md) on how to contribute.
+### Environments
+
+DSO Toolkit development can be done in 3 environments:
+
+#### `fractal`
+The classic environment with Fractal and Stencil. Can only be started from the root of the repository, because it needs two packages (`@dso-toolkit/core` and `dso-toolkit`) running:
+
+```
+yarn fractal
+```
+
+A local webserver is started at http://localhost:43000/. See the [Fractal guide](https://fractal.build/) for more information on Fractal.
```

Comment:
Misschien voor compleetheid hier ook noemen dat Stencil ook hier draait (op http://localhost:53333), dat doe je immers bij de andere development modi ook.

Reply:
Of vind je het hier NOG minder relevant (want: dat is het inderdaad).

Reply:
Nee hoor, ik vind het een goed punt. Ik zal het verwerken